src/containers/offer-page/faq-block/FaqBlock.tsx

```diff
@@ -54,7 +54,7 @@ const FaqBlock = <T extends CreateOrUpdateOfferData>({
   }
 
   const questionsAnswers = data.FAQ.map((el, idx, array) => (
-    <Box key={el.question} sx={styles.faqInputsBlock}>
+    <Box key={idx} sx={styles.faqInputsBlock}>
```

Comment:
There is sonar issue

_Do not use Array index in keys_

Reply:
done